bin/cfgov

```diff
@@ -0,0 +1,73 @@
+#! /usr/bin/perl
+
+my $workDir = $ENV{'CFGOV_HOME'};
+my $currDir = `pwd`;
+
+($workDir eq "") && 
+die("\nPlease set the CFGOV_HOME environment variable to your vagrant directory.\n\n");
+
+chdir($workDir) ||
+ die("\nDirectory $workDir does not exist. Please check your CFGOV_HOME environment variable\n\n");
+
+if ($#ARGV >= 0) {
+	if ($ARGV[0] eq "start") {
+		if (($#ARGV >= 1) && ($ARGV[1] eq "django")) {
+			print("\nStarting Django server on port 8001\n");
+			`vagrant ssh -c "/vagrant/bin/cfgovd start_django" &`;
+		}
+		elsif (($#ARGV >= 1) && ($ARGV[1] eq "watch")) {
+			print("\nStarted gulp watch\n");
+			`vagrant ssh -c "/vagrant/bin/cfgovd start_gulp_watch" &`;
+		}
+		else {
+			print("\nStarting CF Gov Refresh VM...\n\n");
+			my $argList = "";	
+			for ($i=1;$i<=$#ARGV;$i++) {
+				$argList .= $ARGV[$i]." ";
+			}
+			`vagrant up $argList`;
+		}
+		print("\nDone.\n");	
+	}
+	elsif ($ARGV[0] eq 'stop') {
+		if (($#ARGV >= 1) && ($ARGV[1] eq "django")) {
+			print("\nStopping Django server on port 8001\n");
+			for ($i = 0; $i < 3; $i++) {
+				`vagrant ssh -c "/vagrant/bin/cfgovd stop_django" 2>/dev/null`;
+			}
+		}
+		elsif (($#ARGV >= 1) && ($ARGV[1] eq "watch")) {
+			print("\nStopped gulp watch\n");
+			for ($i = 0; $i < 3; $i++) {
+				`vagrant ssh -c "/vagrant/bin/cfgovd stop_gulp_watch" 2>/dev/null`;
+			}
+		}
+		else {
+			print("\nStopping CF Gov Refresh VM...\n\n");
+			`vagrant ssh -c "sudo shutdown -h now"`;
+		}
+		print("\nDone.\n");
+	}
+
+	elsif ($ARGV[0] eq 'init') {
+		print("\nInitializing npm and node js (may take a few minutes)...\n");
+		`vagrant ssh -c "cd /vagrant/bin/; cfgovd init" 2>/dev/null`;
+		print("\nInitializing local node js modules (may take a few minutes)...\n");
+		`vagrant ssh -c "cd /vagrant; sh frontend.sh init" &`;
```

Comment:
Can `init` and `assets` be combined so we're just calling `frontend.sh` and not separating out the methods it contains?